File: Client_Interfaces/python/exemples/mission1.py
from constantList_KP4_1 import *
from pp import *

# Ouverture de la connexion avec le jeu
PP_Open()
# Définition de la position cible
p = PP_Pos (1983.0, 1279.0)
# Récupération de la première unité
u = PP_GetUnitAt(MY_COALITION, 0)
# Ordonner à mon unité courante de se déplacer à la position définie
PP_Unit_ActionOnPosition(u, MOVE, p)
# Fermer la connexion avec le jeu
PP_Close()

File: Client_Interfaces/c-cpp/exemple/mission1.c
#include "PP_Client.h"
#include "../constantList_KP4.7.h"

// déplacer une unité à une position précise
int main (){
	// définition de la position cible
	PP_Pos p1;
	p1.x = 1980.0;
	p1.y = 1676.0;
	// ouverture du jeu
	PP_Open();
	// récupérer la première unité
	PP_Unit u = PP_GetUnitAt(MY_COALITION, 0);
	// ordonner à mon unité courante de se déplacer
	PP_Unit_ActionOnPosition(u, MOVE, p1, 0);
	// fermer le jeu
	PP_Close();
}
